runtime/src/transaction_priority_details.rs

```diff
@@ -28,12 +28,12 @@ pub trait GetTransactionPriorityDetails {
             .process_instructions(
                 instructions,
                 true,  // use default units per instruction
-                false, // stop supporting prioritization by request_units_deprecated instruction
+                true,  // supports prioritization by request_units_deprecated instruction
                 true,  // enable request heap frame instruction
                 true,  // enable support set accounts data size instruction
                        // TODO: round_compute_unit_price_enabled: bool
             )
-            .ok()?;
+            .unwrap_or_default();
```

Comment:
seems like flipping the hardcoded feature gate is the minimal fix?  we should do anything more than that in a follow up to reduce backport exposure

Reply:
It actually makes better sense to keep `ok()?` -- if tx fails on processing compute budget ixs, it'd never be executed anyway, may as well just drop upfront. undoing this line of change...

Reply:
makes sense, thanks!  `.unwrap_or_default()` has not been friendly in the past :wink: